src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -158,9 +166,28 @@ export function fetchFutureAppointments() {
               type: FETCH_PENDING_APPOINTMENTS_SUCCEEDED,
               data: requests,
             });
+
+            const requestSuccessEvent = {
+              event: `${GA_PREFIX}-get-pending-appointments-retrieved`,
+            };
+
+            const expressCareRequests = requests.filter(
+              appt => appt.vaos.isExpressCare,
+            );
+
+            if (vaosExpressCare(getState()) && expressCareRequests.length) {
+              requestSuccessEvent[`${GA_PREFIX}-express-care-number-of-cards`] =
+                expressCareRequests.length;
+            }
```

Comment:
We're only capturing the # of express care cards if express care is enabled and they have more than 1 request, similar to the logic we use to show the tab.